Extensions/quick_tags.js

```diff
@@ -553,7 +553,7 @@ XKit.extensions.quick_tags = new Object({
 
 			if ($(event.target).hasClass("xkit-quick-tags-cp-up") || $(event.target).hasClass("xkit-quick-tags-cp-down")) { return; }
 
-			var m_id = parseInt($(this).attr('data-id'));
+			var m_id = $(this).attr('data-id');
```

Comment:
Okay so technically this code works in either form but this isn't a post id here so it can safely be parseInted. This is just an index into the quick tags tag menu thingy

Reply:
hmmmmmmmmm i would rather not touch extensions that are not broken so i will undo this

Reply:
nvm cant be bothered \*tf2 scout voice\* PUUUUUUSH